addons/l10n_latam_invoice_document/wizards/account_debit_note.py

```diff
@@ -14,3 +14,11 @@ def create_debit(self):
             new_move._compute_l10n_latam_document_type()
             new_move._onchange_l10n_latam_document_type_id()
         return res
+
+    def _prepare_default_values(self, move):
```

Comment:
The lines 14-15 are there for a reason.  (so I wonder if we can do in the same way by putting a correct document type first as it could be taken care of by the compute (have not we done something like it before? (is this only for 16?))

Reply:
Good point Jos. I decided to leave it because the _onchange_l10n_latam_document_type_id method executes a _compute_name and I didn't know what impact it could have in other localizations. 
But in fact for the argentinian localization you can remove all inheritance to the method. 
I remove it so we can evaluate if it affects elsewhere.

The bug is also in 15. I can reassign the PR to 15 or start a new . As you prefer

Reply:
Maybe in master.  I meant the opposite.

Reply:
@jco-odoo  I created a new PR because I needed to keep the old one at 16.0 

https://github.com/odoo/odoo/pull/154206